Resources/Textures/DeltaV/Objects/Tools/triangle_spice.rsi/meta.json

```
@@ -0,0 +1,79 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Leonardo DaBepis#4869 on discord / @leonardo-dabepis on Tumblr",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "welder_flame"
+    },
+    {
+      "name": "inhand-left"
+    },
+    {
+      "name": "inhand-right"
+    },
+    {
+      "name": "inhand-left-flame",
+      "directions": 4,
+      "delays": [
+        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
+        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
+        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
+        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ]
+      ]
+    },
+    {
+      "name": "inhand-right-flame",
+      "directions": 4,
+      "delays": [
+        [
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13
+        ],
+        [
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13
+        ],
+        [
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13
+        ],
+        [
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13,
+          0.13
+        ]
```

Oh I didn't see this one
```suggestion
        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ],
        [ 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13, 0.13 ]
```